sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -3211,6 +3222,8 @@ export const StorageOAuthScopes: string | string[];
 
 // @public
 export interface StoragePipelineOptions {
+    // (undocumented)
+    audience?: BlobAudience;
```

Comment:
can we use just `string`?  for reference this PR shows the pattern we use in Form Recognizer: https://github.com/Azure/azure-sdk-for-js/pull/20522/files#diff-47c5d6a4cd99cdff5e5c345a6df937952d545cbd0da40274774a21f9378ff1b2

Reply:
Updated